config/gtfs-feeds.csv

```diff
@@ -6,7 +6,7 @@ DELFI;CC BY 4.0;DELFI;nein;https://de.data.public-transport.earth/gtfs-germany.z
 DING;dl-de/by-2.0 / Shapes ODbL;Datensatz der NVBW GmbH, Shapes (C) OpenStreetMap Mitwirkende;Ja;https://www.nvbw.de/fileadmin/user_upload/service/open_data/fahrplandaten_mit_liniennetz/ding.zip;https://www.nvbw.de/open-data/fahrplandaten/fahrplandaten-mit-liniennetz;opendata@nvbw.de;Nein
 FlixBus;?;?;ja;https://data.ndovloket.nl/flixbus/flixbus-eu.zip;https://data.ndovloket.nl/flixbus/;;Nein
 HNV;dl-de/by-2.0;Datensatz der NVBW GmbH, Shapes (C) OpenStreetMap Mitwirkende;Ja;https://www.nvbw.de/fileadmin/user_upload/service/open_data/fahrplandaten_mit_liniennetz/hnv.zip;https://www.nvbw.de/open-data/fahrplandaten/fahrplandaten-mit-liniennetz;opendata@nvbw.de;Nein
-HVV;dl-de/by-2.0;Hamburger Verkehrsverbund GmbH;Nein;https://daten.transparenz.hamburg.de/Dataport.HmbTG.ZS.Webservice.GetRessource100/GetRessource100.svc/40c90bad-79e6-4f03-ac33-8f1f7095f7a1/Upload__HVV_Rohdaten_GTFS_Fpl_20220408.zip;https://suche.transparenz.hamburg.de/?q=GTFS&esq_not_all_versions=true&sort=publishing_date+desc%2Ctitle_sort+asc&esq_not_all_versions=true;;Nein
+HVV;dl-de/by-2.0;Hamburger Verkehrsverbund GmbH;Nein;https://scraped.data.public-transport.earth/de/hvv-gtfs.zip;https://suche.transparenz.hamburg.de/?q=gtfs&esq_not_all_versions=true&sort=publishing_date+asc%2Ctitle_sort+asc&esq_not_all_versions=true;;Nein
```

Comment:
Shall I link to [Julius' scraper readme linking to the HVV portal](https://github.com/juliuste/public-transport-data-scraper/tree/f2529284f76a00bf1a01a07e5ed9a9ee4d043fa4#scraped-datasets) instead?